dev-vcs/git-cola/git-cola-4.8.2.ebuild

```diff
@@ -14,14 +14,14 @@ SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="~amd64 ~x86"
+KEYWORDS="~amd64"
```

Comment:
dev-python/PyQt6 has just been ~x86 keyworded, so you should be able to keep this.

Reply:
Seems there still some issues with ~x86

Reply:
what issues?

Reply:
pkgcheck complanied about QtPy, but seems it was even stable on x86, so I don't really understand. In any case, I think x86 was being slowly deprecated, so didn't wasted too much investigating it.

Reply:
If you depended on PyQt6 via QtPy[pyqt6], but did not rebase this branch onto a recent pull of origin/master, then failure on x86 would have been clear. It was unmasked only a couple of weeks ago on that branch (and keyworded only shortly before).